content/docs/welcome-guide/setup/installing-linux.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Setting up O3DE from a deb package in Linux
+description: Learn how to install and set up Open 3D Engine (O3DE) using a deb package in Linux.
+weight: 200
+draft: true
+---
+
+To get started quickly with O3DE in Linux, download and install the deb package. The following instructions guide you through the installation process. After a successful install, you'll have a stable, pre-built version of the engine and its Gems, and you'll be ready to create new or open existing projects with the **Project Manager** tool.
+
+## Prerequisites
+
+The instructions here assume that you have met all hardware and software requirements listed in [O3DE System Requirements](../requirements#linux).
+
+## Installing O3DE
+
+1. Get the latest version of the deb package from the [O3DE download](https://o3de.org/download/) page.
+
+1. Run the installer from your download location.
+```shell
+sudo apt install <path_to_deb_package>/<debian_package_name>.deb
+```
+Where:
+* `<path_to_deb_package>` is the path where the deb package was downloaded to, e.g. `~/Downloads`
+* `<debian_package_name>` is the name of the package, e.g. `O3DE_2111_1.deb`
+
+1. The default install location is: `/opt/O3DE/<version>`. Where `<version>` is the version of the installer, e.g. `2111_1`.
```

Comment:
not really, they can unpack the deb package into a different location, but that is a manual process (involves using dpkg). Its likely also going to be confusing because the extracted path will have that `/opt` folder